zerver/data_import/mattermost.py

```diff
@@ -325,7 +360,7 @@ def fix_mentions(content: str, mention_user_ids: Set[int]) -> str:
             recipient_id=recipient_id,
             rendered_content=rendered_content,
             topic_name=topic_name,
-            user_id=user_id,
+            user_id=sender_user_id,
```

Comment:
This bit looks like a simple pure refactor to remove an unnecessary variable; extract it to a prep commit?

Reply:
Done.